client/quasar.conf.js

```diff
@@ -7,7 +7,7 @@ module.exports = function(ctx) {
     // --> boot files are part of "main.js"
     // https://quasar.dev/quasar-cli/cli-documentation/boot-files
     boot: ["i18n", "axios"],
-    preFetch: true,
+    preFetch: false,
```

Comment:
Just curious, was there a specific failure that changing this fixed?

Reply:
When I saw this, I thought it was an artifact of a prior merge. I'll look into it.

Reply:
I think it's related to SSR. I was just curious more than anything.

Reply:
You're correct. https://quasar.dev/quasar-cli/prefetch-feature#Installation

Here's where I made the change: https://github.com/MESH-Research/CCR/commit/b9331279e569e1eb275d401314d5bd0be3e4e34f